### PR TITLE
authoring changes for mini cards

### PR DESCRIPTION
**Pull request by skumawat1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix[ #<gh-issue-id>](https://adobe-acs-group.atlassian.net/browse/WRLDBAEMS-106)

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/en/drafts/skumawat/mini-cards
- After: https://mini-cards--world-bank--aemsites.hlx.live/en/drafts/skumawat/mini-cards
- 
image

### CODE REVIEW

File: component-models.json [hunk 1]
```diff
@@ -611,5 +611,65 @@
...
+        "label": "Image",
+        "multi": false
+      },
+      {
+        "component": "aem-tag",
+        "name": "mini-card-tag",
+        "label": "Card Type",
+        "valueType": "string",
+        "required": true,
+        "rootPath": "/content/cq:tags/world-bank",
```
Comment (mkbansal1) [inline component-models.json:631]: set root path as /content/cq:tags/world-bank/content-type

File: component-models.json [hunk 2]
```diff
@@ -611,5 +611,65 @@
...
+        "multi": false
+      },
+      {
+        "component": "text",
+        "valueType": "string",
+        "name": "mini-card-title",
+        "value": "",
+        "label": "Title",
```
Comment (mkbansal1) [inline component-models.json:639]: make it required

File: component-models.json [hunk 3]
```diff
@@ -611,5 +611,65 @@
...
+        "valueType": "string",
+        "name": "mini-card-location",
+        "value": "",
+        "label": "Location",
+        "description": "Sets the description of the Card"
+      },
+      {
+        "component": "text",
+        "name": "curated-card-link",
+        "value": "",
```
Comment (mkbansal1) [inline component-models.json:669]: make it required

File: component-models.json [hunk 4]
```diff
@@ -611,5 +611,65 @@
...
+        "label": "Time",
+        "description": "Sets the Time of the Card"
+      },
+      {
+        "component": "text",
+        "valueType": "string",
+        "name": "mini-card-location",
+        "value": "",
+        "label": "Location",
+        "description": "Sets the description of the Card"
```
Comment (mkbansal1) [inline component-models.json:664]: remove descriptions

**Review by mkbansal1 — CHANGES_REQUESTED** [review]

File: component-models.json (hunk 3 above)
Comment (skumawat1) [reply component-models.json:666]: done

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**